src/components/UI/Display/Apps/AppDetailNew/AppDetailPage.tsx

```diff
@@ -326,7 +326,8 @@ const AppDetail: React.FC<IAppDetailPageProps> = (props) => {
             <AppDetailPageOptionalValue value={props.website}>
               {(value) => (
                 <a href={value} target='_blank' rel='noopener noreferrer'>
-                  {value}
+                  {value}&nbsp;
```

Comment:
```suggestion
                  {value}{' '}
```

Reply:
Will this prevent the icon from breaking into a separate line if the URL is long?

Reply:
I don't know about that, but `{' '}` is what we use in other parts of the code base

Reply:
ACK

Reply:
`{' '}` is a normal space, so it won't prevent it from breaking into a separate line. `&nbsp;` is a non-breaking space, which will prevent it.

alternatively you can make some changes to the CSS to adjust how whitespace is handled. Would have to Google exactly what property though, never know it off the top of my head